docs/developer/advanced_features.rst

```diff
@@ -18,4 +18,26 @@ Morison Element
 .. TO DO: add passive yaw
 .. TO DO: add morison element
 .. TO DO: add passive yaw
-.. TO DO: add PTO-Sim
\ No newline at end of file
+.. TO DO: add PTO-Sim
+
+Extract Mask Variables
+----------------------
+The Simulink variable workspace is inaccesible in the MATLAB workspace by default.
+Simulink variables can be imported to the MATLAB workspace using the block handle 
+of the Simulink block being probed. The block parameters can be used by developers
+to programmatically set block parameters by being able to access the unique tags
+that Simulink uses for a particular block parameter. This is also useful for the code
+initialization of Simulink mask blocks.
+The function :code:`ExtractMaskVariables()` 
+located in :code:`source\functions\simulink\mask`, can be used to extract the block
+parameters in following steps,
+
+* Open the pertinent Simulink model,
+* Run the function with the address of the block being probed as the argument,
+* Explore the :code:`mask' data structure in the MATLAB workspace.
```

Comment:
@salhus, `mask` needs to be delimited by two `` ` ``s, i.e. `` :code:`mask` `` not `` :code:`mask' ``.

Reply:
From [this line](https://github.com/WEC-Sim/WEC-Sim/pull/958/checks#step:5:41) in the build logs.

Reply:
@H0R5E Thanks so much Matthew! I appreciate the help!